### DISCUSSION TITLE
Sensor Installation

### DISCUSSION THREAD

**Discussion by Shorider** [Security-Onion-Solutions/securityonion#6160] [General]
Good morning everyone,

I've been running into this issue for the past 4-5 days and I'm not sure how to move forward.  I'm trying to set up a forward sensor.  I'm installing via security onion via the following:

git clone https://github.com/Security-Onion-Solutions/securityonion
cd securityonion
sudo bash so-setup-network

I run through the options to set it up and when it starts to install, it just hangs at installing docker around 8% and just sits there until it errors out.  I have tried all the configuration changes I can think of to this point and have gotten nowhere.  I have verified internet connectivity through pinging both github and google.  Am I missing something?

Thank you all for your time!

**Comment by TOoSmOotH** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/1720
